distributed/deploy/tests/test_adaptive.py

```diff
@@ -19,43 +20,6 @@
 )
 
 
-@pytest.mark.asyncio
-async def test_simultaneous_scale_up_and_down(cleanup):
```

Comment:
This test was added in https://github.com/dask/distributed/pull/1608 to disallow simultaneous up-/down-scaling but the current implementation doesn't work at all since the API changed. Instead it only raises error logs